workspaces/src/rpc/client.rs

```diff
@@ -170,19 +170,15 @@ impl Client {
     pub(crate) async fn view_state(
         &self,
         contract_id: AccountId,
-        prefix: Option<StoreKey>,
-    ) -> anyhow::Result<HashMap<String, Vec<u8>>> {
-        self.view_state_raw(contract_id, prefix, None)
-            .await?
-            .into_iter()
-            .map(|(k, v)| Ok((String::from_utf8(k)?, v.to_vec())))
-            .collect()
+        prefix: Option<&[u8]>,
+    ) -> anyhow::Result<HashMap<Vec<u8>, Vec<u8>>> {
+        self.view_state_raw(contract_id, prefix, None).await
     }
```

Comment:
we can probably just delete this now, since it's the same as `view_state_raw` and rename `view_state_raw` as `view_state`

Reply:
Sure! They are slightly different APIs so I kept, but I'll remove now since all internal